Djikstra/Djikstra/Graph.cpp

```diff
@@ -0,0 +1,159 @@
+#include <vector>
+#include <queue>
+#include <deque>
+#include <map>
+#include <fstream>
+#include <string>
+#include <stdexcept>
+
+
+#include "Graph.h"
+
+
+namespace Graph
+{
+    class Comparator {
+    public:
+        bool operator()(const Vertex* v1, const Vertex* v2)
+        {
+            return v1->distance > v2->distance;
+        }
+    };
+
+
+
+    void Graph::vertexes_exist(const std::string& name1, const std::string& name2)
```

Comment:
This should be named like `assert_vertices_exist`, since what we do here is an _assertion_.